llvm/lib/Target/ARM/AsmParser/ARMAsmParser.cpp

```diff
@@ -1347,6 +1368,14 @@ class ARMOperand : public MCParsedAsmOperand {
   bool isRegListWithAPSR() const {
     return Kind == k_RegisterListWithAPSR || Kind == k_RegisterList;
   }
+  bool isDReg() const {
+    return isReg() &&
+           ARMMCRegisterClasses[ARM::QPRRegClassID].contains(Reg.RegNum);
```

Comment:
This looks identical to `isQReg` below – it still checks against `QPRRegClassID`. Shouldn't it have a different id?

Reply:
Whoops, thank you! I am surprised this didn't catch any tests.